src/sentry/runner/commands/devserver.py

```diff
@@ -232,11 +237,15 @@ def devserver(reload, watchers, workers, experimental_spa, styleguide, prefix, e
     if styleguide:
         daemons += [("storybook", ["./bin/yarn", "storybook"])]
 
+    daemons_to_skip = set(skip.split(",")) if skip else ()
     cwd = os.path.realpath(os.path.join(settings.PROJECT_ROOT, os.pardir, os.pardir))
 
     manager = Manager(Printer(prefix=prefix))
     for name, cmd in daemons:
-        manager.add_process(name, list2cmdline(cmd), quiet=False, cwd=cwd)
+        cmdline = list2cmdline(cmd)
+        if name not in daemons_to_skip:
+            click.echo("Adding daemon '{}' with command: {}".format(name, cmdline))
```

Comment:
Appropriate logging? I thought this would help, both in identifying the daemon names that are valid arguments to the `--skip` option and in providing commands for the developer to emulate in launching the replacement processes.

Because it's not being echoed from within the Honcho manager, the log lacks the normal prefix and looks a little weird.

Reply:
They do get echoed:

```
22:43:15 system  | webpack started (pid=82959)
22:43:15 system  | ingest started (pid=82960)
22:43:15 system  | server started (pid=82961)
```

Reply:
Right. I was leaning more verbose than that, but if I make those changes to the help text then it's probably not necessary.

Reply:
Yes, I agree that it wouldn't be necessary.